src/main/java/com/ericsson/eiffel/remrem/semantics/SemanticsService.java

```diff
@@ -61,6 +110,7 @@ public String generateMsg(String msgType, JsonObject bodyJson){
 
         Event event = createEvent(eventNodes, eventType);
         event.generateMeta(msgType, msgNodes);
+        event.setMeta(event.meta);
```

Comment:
row 101 to 113 above should be placed in a method so that we can create create Event objects form json strings for other purposes than publishing. A static method would be preferable.